src/joint_moteus.cpp

```diff
@@ -0,0 +1,294 @@
+/**
+ * @file joint_moteus.cpp
+ * @author J. Diego Caporale (jdcap@seas.upenn.edu), Lucien Peach 
+ * (peach@seas.upenn.edu), Shane Rozen-Levy (srozen01@seas.upenn.edu)
+ * @brief Moteus powered joint class implementation
+ * @date 2023-07-13
+ * 
+ * @copyright BSD 3-Clause License, Copyright (c) 2021 The Trustees of the University of Pennsylvania. All Rights Reserved
+ * 
+ */
+
+
+#include "kodlab_mjbots_sdk/joint_moteus.h"
+
+namespace kodlab{
+namespace mjbots{
+
+// kDefaultQuery defintion
+const ::mjbots::moteus::QueryCommand JointMoteus::kDefaultQuery{
+  ::mjbots::moteus::Resolution::kInt8,
```

Comment:
I feel like we should comment this so its readable to the user.

Reply:
Once these are commented I will want to give this a look over with Diego to make sure we are happy with the standards.